tsconfig.json

```diff
@@ -0,0 +1,76 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
```

Comment:
We may want to remove the commented-out options before we merge, for readability? (Or at least once we've settled on a good config.)

Reply:
yah i will do that when i get this working properly